sdk/core/azure-core/azure/core/pipeline/transport/_base.py

```diff
@@ -149,7 +149,7 @@ def open(self):
     def close(self):
         """Close the session if it is not externally owned."""
 
-    def sleep(self, duration):  # pylint: disable=no-self-use
+    def sleep(self, duration: float) -> None:  # pylint: disable=no-self-use
```

Comment:
pylint's no-self-use is an optional check these days, no? We should be able to remove all these suppressions I think (they were, IMHO, always 90% more annoying than helpful - and our guidelines actually discourage use of static methods, which means that the rule was in conflict with our guidelines...)

Reply:
Done, so solving this conversation